strict-mvar/strict-mvar.cabal

```
@@ -0,0 +1,43 @@
+cabal-version:       3.0
+name:                strict-mvar
+version:             0.1.0.0
+synopsis:            Strict MVars for `IO` and `io-sim`
+description:         Strict MVars for `IO` and `io-sim`.
+license:             Apache-2.0
+license-files:
+  LICENSE
+  NOTICE
+copyright:           2019-2023 Input Output Global Inc (IOG).
+author:              IOHK Engineering Team
+maintainer:          operations@iohk.io
+category:            Control
+build-type:          Simple
+tested-with:         GHC == 8.10.7, GHC == 9.2.5, GHC == 9.4.3
+
+source-repository head
+  type:     git
+  location: https://github.com/input-output-hk/io-sim
+  subdir:   strict-mvar
+
+flag asserts
+  description: Enable assertions
+  manual:      False
+  default:     False
+
+library
+  hs-source-dirs:      src
+
+  exposed-modules:     Control.Concurrent.Class.MonadMVar.Strict
+  default-language:    Haskell2010
+  build-depends:       base        >= 4.9 && <4.18,
+                       io-classes  >= 0.3 && <0.5
```

we're about to publish `0.5`, so
```suggestion
                       io-classes  >= 0.3 && <0.6
```